.github/workflows/ci.yml

```diff
@@ -80,10 +80,10 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install npm dependencies
-        run: npm run ci --if-present
+        run: npm run install --if-present
```

Comment:
Change this to just `npm ci`
Don't use --if-present for this since it means it will silently fail and cause the following steps to not work

Reply:
The `--if-present` returns 0(True) only if the task/script is not there
If the task/script is there, then it will run as normal
The I plan to remove them once we get accustomed to the workflow